cachi2/core/package_managers/pip.py

```diff
@@ -1190,8 +1190,8 @@ def __init__(self) -> None:
         # The package name as defined in the requirement line
         self.raw_package: str = ""
         self.extras: set[str] = set()
-        self.version_specs: list[str] = []
-        self.environment_marker: str = ""
+        self.version_specs: list[tuple[str, str]] = []
+        self.environment_marker: Optional[str] = None
```

Comment:
How are the 2 hunks in this patch related? Is there a direct relationship? Unless there IS a connection, this should be squashed to commit https://github.com/containerbuildsystem/cachi2/pull/637/commits/6d10d8f85f1d2e9be7c95a789471d7814d1114af and commit message should be adjusted.